server/src/main/java/com/hedera/block/server/verification/signature/SignatureVerifier.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.server.verification.signature;
+
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+
+/**
+ * An interface for verifying signatures.
+ */
+public interface SignatureVerifier {
+
+    /**
+     * Verifies the signature of a hash.
+     *
+     * @param hash the hash to verify
+     * @param signature the signature to verify
+     * @return true if the signature is valid, false otherwise
+     */
+    Boolean verifySignature(Bytes hash, Bytes signature);
```

Comment:
could add the non-null annotation to the parameter for the static code analysis

Reply:
✅  Done!